Edit column names in bigquery table.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
```

Comment:
May be add some context here about why we need this procedure. Reading this document it looks like we cannot simply rename fields or tables in big query? is this procedure for solving this restriction?

Reply:
Great idea @kashifpk
I Did this.
https://github.com/NebraLtd/hm-diag/blob/0fe933265d2af405f6ae204e470fff3469f716f1/Edit%20column%20names%20in%20bigquery%20table.md?plain=1#L1-L7